travis-ci/packer-templates
# Ci ubuntu 1804

## What is the problem that this PR is trying to fix?
To add template to support bionic 
## What approach did you choose and why?
Create new build template and update relevant cookbooks
## How can you test this?
* build from local env, result as expected 
* build by pack-build, result as expected
## What feedback would you like, if any?
Appreciate any feedback.
